docs/conf.py

```diff
@@ -172,9 +173,9 @@
 
 
 html_theme_options = {
-    "logotext1": "tardis",  # white,  semi-bold
-    "logotext2": "",  # orange, light
-    "logotext3": ":docs"   # white,  light
+#    "logotext1": "tardis",  # white,  semi-bold
+#    "logotext2": "",  # orange, light
+#    "logotext3": ":docs"   # white,  light
```

Comment:
Why commented and not deleted?

Reply:
I don't know what the point of these were, so I figured that I would leave them in to show what someone tried to do for people who may want to develop that part of the code in the future.

Reply:
@epassaro do you know what these were supposed to do?

Reply:
Not sure. Is this related to any warning?

Reply:
Yes. It tells us that "logotext1" etc are not supported keys. Was there a specific reason that you added them in?

Reply:
`conf.py` is a file from the astropy package template, I didn't wrote it. Comment it out if that's ok, try not deleting anything.